stable/influxdb/values.yaml

```diff
@@ -114,6 +114,11 @@ ingress:
   tls: false
   # secretName: my-tls-cert # only needed if tls above is true
   hostname: influxdb.foobar.com
+
+  ## Ingress annotations done as key:value pairs
+  ## For a full list of possible ingress annotations, please see
+  ## ref: https://github.com/kubernetes/ingress-nginx/blob/master/docs/user-guide/nginx-configuration/annotations.md
```

Comment:
I don't think this paragraph is necessary imho. Furthermore the link is just for nginx-ingress annotations, not applicable to other ingress controllers.

Reply:
Thanks. Solved.